src/ui/public/vislib/lib/axis.js

```diff
@@ -98,7 +97,7 @@ export default function AxisFactory(Private) {
 
     adjustSize() {
       const self = this;
-      const config = this.config;
+      const config = this.axisConfig;
```

Comment:
👍 when we want to shorten `axisConfig` or `visConfig` to `config`, I think this is exactly how we should do it; local to a function somewhere, but close enough that it's easy to trace back to it's name